docker-compose.yml

```diff
@@ -1,15 +1,80 @@
 version: "3.1"
 
 services:
+  conseil:
+    image: cryptonomictech/conseil:0.2002.0020-SNAPSHOT
+    ports:
+      - 80:80
+    environment:
+#  List of supported environment variables:
+#  DBHOST=${DB_Host:=db}
+#  DBUSER=${DB_User:=user}
+#  DBPW=${DB_Password:=password}
+#  DBDB=${DB_Database:=conseil}
+#  DBPORT=${DB_Port:=5432}
+#  XTZSCHEME=${XTZ_Scheme:=http}
+#  XTZHOST=${XTZ_Host:=node}
+#  XTZPREFIX=${XTZ_Prefix:=}
+#  XTZPORT=${XTZ_Port:=8732}
+#  XTZNET=${XTZ_Network:=mainnet}
+#  APIPORT=${API_PORT:=80}
+#  APIKEY=${API_KEY:=conseil}
+      DB_Host: conseil-postgres
+      DB_User: conseiluser
+      DB_Password: p@ssw0rd
+      DB_Database: conseil-local
+      DB_Port: 5432
+      XTZ_Scheme: https
+      XTZ_Host: tezos-staging.cryptonomic-infra.tech
+      XTZ_Port: 443
+      XTZ_Network: mainnet
+    entrypoint: ["/root/wait-for.sh", "conseil-postgres:5432", "-t", "120", "--", "/root/entrypoint.sh", "conseil"]
+    volumes:
+      - "conseil-sql:/root/sql/"
+    networks:
+      - conseil-net
+
+  lorre:
+    image: cryptonomictech/conseil:0.2002.0020-SNAPSHOT
+    environment:
+      DB_Host: conseil-postgres
+      DB_User: conseiluser
+      DB_Password: p@ssw0rd
+      DB_Database: conseil-local
+      DB_Port: 5432
+      XTZ_Scheme: https
+      XTZ_Host: tezos-staging.cryptonomic-infra.tech
+      XTZ_Port: 443
+      XTZ_Network: mainnet
+    command: lorre
+    depends_on:
+      - conseil-postgres
+    networks:
+      - conseil-net
+
   conseil-postgres:
     image: postgres:11.6
     ports:
-    - 5432:5432
+      - 5432:5432
     environment:
       POSTGRES_USER: "conseiluser"
       POSTGRES_PASSWORD: "p@ssw0rd"
       POSTGRES_DB: "conseil-local"
       POSTGRES_INITDB_ARGS: "--lc-collate=en_US.UTF-8 -E UTF8"
     volumes:
-    - "./pgdata:/var/lib/postgresql/data"
-    - "./sql/conseil.sql:/docker-entrypoint-initdb.d/conseil.sql"
+#     uncomment if you want your data to be stored in the local directory
+#      - "./pgdata:/var/lib/postgresql/data"
```

Comment:
Changing default, will impact expectations set by the /README.md file, under the paragraph on testing locally using a docker postgres with compose.
I suggest changing that paragraph too, to say that you need to modify the compose file to keep data persistent between restarts.

As a side-note, I think I've lost the reason why we have two different readme files for project-root and `docs` folder? Some information there is overlapping and can easily get inconsistent, can't we find a way to merge them or at least make one refer to the other for details?

Reply:
no, this data stays until you remove container with `docker-compose rm`. Now when clearing the data I always run `docker-compose rm` and I delete `/pgdata` directory. With this change you'll only need to `docker-compose rm` after you stop the container.

Reply:
I didn't know this, are you saying that if I `docker-compose down`, using the file as in this PR, I would find all the data back when I `docker-compose up` the db?

Reply:
I don't usually use `-d` parameter, so I use `Ctrl + C` to stop the container.  But you're right `docker-compose down` stops and removes container. Instead try using `docker-compose stop`.

Reply:
Ok, got it